src/common/modules/data/DefaultSettings.js

```diff
@@ -11,12 +11,22 @@
  * @const
  * @type {Object}
  */
+
+// checks for OS dark theme and returns the appropriate background color
+ const setQrBackgroundColor = () => {
+     if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        return "#d7d7db"; // Photon Grey 30
+     } else {
+        return "ffffff";
+     }
+ };
+
 const defaultSettings = Object.freeze({
     debugMode: false,
     popupIconColored: false,
     qrCodeType: "svg",
     qrColor: "#0c0c0d",
```

Comment:
Also generally, I guess you could maybe also adjust the QR code color in case of "dark mode", so that it matches to the background color? Or is this really not needed? (did you check the contrast/tested with QR code scanners etc.)

Reply:
I tried out different combinations, always testing for contrast and QR scannability (if that's a word?). This combination seemed to reduce the white glare, stay in line with Photon and stay scannable better than others. But there might be a better solution